pkg/config/kds/config.go

```diff
@@ -0,0 +1,41 @@
+package kds
+
+import (
+	"time"
+
+	"github.com/pkg/errors"
+	"go.uber.org/multierr"
+
+	"github.com/Kong/kuma/pkg/config"
+)
+
+func DefaultKumaDiscoveryServerConfig() *KumaDiscoveryServerConfig {
+	return &KumaDiscoveryServerConfig{
+		GrpcPort:        5685,
+		RefreshInterval: 1 * time.Second,
+	}
+}
+
+// Kuma Discovery Service (KDS) server configuration.
+type KumaDiscoveryServerConfig struct {
+	// Port of a gRPC server that serves Kuma Discovery Service (KDS).
+	GrpcPort uint32 `yaml:"grpcPort" envconfig:"kuma_discovery_server_grpc_port"`
+
+	// Interval for refreshing state of the world
+	RefreshInterval time.Duration `yaml:"refreshInterval" envconfig:"kuma_discovery_server_refresh_interval"`
+}
+
+var _ config.Config = &KumaDiscoveryServerConfig{}
+
+func (c *KumaDiscoveryServerConfig) Sanitize() {
+}
+
+func (c *KumaDiscoveryServerConfig) Validate() (errs error) {
+	if 65535 < c.GrpcPort {
```

Comment:
nit: <=

Reply:
That's confusing to write a number on the left side. But if `c.GrpcPort` is equal to 65535 then it is not an error. I just copied that Validation from `config/mads/config.go`

Reply:
Oh right, I agree with confusion of writing the number of left side. Change it if you want. I find `c.GrpcPort > 65535` more readable